relay-filter/src/config.rs

```diff
@@ -210,6 +224,13 @@ pub struct FiltersConfig {
     /// Configuration for the releases filter.
     #[serde(default, skip_serializing_if = "ReleasesFilterConfig::is_empty")]
     pub releases: ReleasesFilterConfig,
+
+    /// Configuration for the healthcheck filter.
+    #[serde(
+        default,
+        skip_serializing_if = "HealthCheckEndpointsFilterConfig::is_empty"
+    )]
+    pub health_check: HealthCheckEndpointsFilterConfig,
```

Comment:
This filter works for any pattern, not just health checks. Could we rename it to reflect it (e.g. `transaction_names`)? It's ok if we just use it for health checks, but once we introduce `health_check` we should keep it for backward compatibility.